src/test/scala/com/campudus/tableaux/api/content/CellLevelAnnotationsTest.scala

```diff
@@ -337,4 +337,67 @@ class CellLevelAnnotationsTest extends TableauxTestBase {
       }
     }
   }
+
+  @Test
+  def addAnnotationWithoutLangtagsToConcatColumn(implicit c: TestContext): Unit = {
```

Comment:
Do we need `addAnnotationWithLangtagsToConcatColumn` as well? Otherwise this could be renamed to `addAnnotationToConcatColumn`.

Reply:
Will do